dotnet-engine/Yggdrasil.Engine.Tests/YggdrasilEngineTest.cs

```diff
@@ -77,7 +77,7 @@ public void TestClientSpec()
                 var toggleName = (string)test["toggleName"];
                 var expectedResult = (bool)test["expectedResult"];
 
-                var result = yggdrasilEngine.IsEnabled(toggleName, context) ?? false;
+                var result = yggdrasilEngine.IsEnabled(toggleName, context!) ?? false;

```

Comment:
Just a test, this can never be null or the test will explode, telling the compiler to hush here feels okay